Headlines/src/Controllers/NewsTableViewController.swift

```diff
@@ -13,7 +13,8 @@ import ViewAnimator
 
 class NewsTableViewController: UITableViewController,
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/e02648ce2406a2f85d493d64af3d34cc6c9bd946/Headlines/src/Controllers/NewsTableViewController.swift#L14"></span>Type body should span 200 lines or less excluding comments and whitespace: currently spans 262 lines</td>
    </tr>
  </tbody>
</table>